war_or_peace_runner_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rspec'
+require './lib/card'
+require './war_or_peace_runner'
+
+RSpec.describe Game do 
```

Comment:
This file would be `game_spec.rb` after refactoring the `Game` class into it's own file.

Runner files do not need testing.